apps/src/applab/designElements/ColorPickerPropertyRow.jsx

```diff
@@ -40,7 +40,7 @@ var ColorPickerPropertyRow = React.createClass({
   handleColorChange: function (color) {
     if (color.rgb.a == 1) {
       // no transparency set
-      this.changeColor('#' + color.hex);
+      this.changeColor(color.hex);
```

Comment:
See https://github.com/casesandberg/react-color/issues/51.  Although `react-color` is supposed to return hex values with a leading `#` since 1.3.8, there's a bug in the published npm 2.0.0 version where the transpiled lib is out of date and returns hex colors without the leading `#`.  Our fork actually has the expected behavior and returns hex with leading `#` so we don't need to add it here anymore.